src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/get_file_request.cc

```diff
@@ -184,8 +176,18 @@ void GetFileRequest::Run() {
   }
 
   // Handle offset
-  LOG4CXX_DEBUG(logger_, "Handle offset parameter");
+  LOG4CXX_DEBUG(logger_, "Handle offset and length parameters");
   const uint64_t file_size = file_system::FileSize(full_path);
+
+  if ((*message_)[strings::msg_params].keyExists(strings::length)) {
+    length_ = (*message_)[strings::msg_params][strings::length].asInt();
```

Comment:
`std::min<(*message_)[strings::msg_params][strings::length].asInt(), length_ - offset_>`

Reply:
Again from what I understand the GetFile would return the bytes between `offset` and `offset+length` so not sure what I would need this for.